boto/endpoints.json

```diff
@@ -387,21 +401,20 @@
         "eu-central-1": "storagegateway.eu-central-1.amazonaws.com"
     },
     "sts": {
-        "ap-northeast-1": "sts.amazonaws.com",
-        "ap-southeast-1": "sts.amazonaws.com",
-        "ap-southeast-2": "sts.amazonaws.com",
+        "ap-northeast-1": "sts.ap-northeast-1.amazonaws.com",
```

Comment:
This would be a breaking change because this regionalized endpoints require customer opt in.